### PR TITLE
Pull latest from Master for AIP additional evid branch

### PR DESCRIPTION
**Pull request by sajjadh-v1**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: webpack/webpack.common.js [hunk 1]
```diff
@@ -5,80 +5,80 @@ const MiniCSSExtractPlugin = require('mini-css-extract-plugin');
...
+            rules: [
+                {
+                    test: /\.ts$/,
+                    use: [
+                        'ts-loader',
+                    ]
+                }
+            ]
+        },
 };
```
Comment (augustio) [inline webpack/webpack.common.js:30]: The diff here is about the size of indentation? Is this needed?

File: webpack/webpack.dev.js [hunk 2]
```diff
@@ -2,13 +2,13 @@ const merge = require('webpack-merge');
...
+    devtool: 'inline-source-map',
 };
 
 const clientConfig = {
-  mode: "development",
-  devtool: 'inline-source-map',
+    mode: "development",
+    devtool: 'inline-source-map',
 
 };
```
Comment (augustio) [inline webpack/webpack.dev.js:14]: The diff here is about the size of indentation? Is this needed?

File: webpack/webpack.common.js [hunk 3]
```diff
@@ -5,80 +5,80 @@ const MiniCSSExtractPlugin = require('mini-css-extract-plugin');
...
+        plugins: [
+            new CopyWebpackPlugin([
+                {from: path.resolve('node_modules/govuk-frontend/govuk/assets/'), to: 'assets'},
+                {from: path.resolve('app/assets/images/'), to: 'assets/images'}
+            ]),
+            new MiniCSSExtractPlugin({
+                filename: '[name].css'
+            })
+        ],
 };
```
Comment (augustio) [inline webpack/webpack.common.js:82]: The diff here is about the size of indentation? Is this needed?

File: webpack/webpack.common.js (hunk 1 above)
Comment (sajjadh-v1) [reply webpack/webpack.common.js:30]: Not needed but all these formatting changes are auto merged

**Review by augustio — APPROVED** [review]
LGTM.

**Review by gregorio-massara — DISMISSED** [review]
LGTM

**Review by gregorio-massara — APPROVED** [review]
LGTM

**Review by augustio — APPROVED** [review]
LGTM.

**Pull request merged**